content/numpy/concepts/built-in-functions/terms/trace/trace.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: '.trace()'
+Description: 'Calculates the sum of the elements along the main diagonal of an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'Matrices'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.trace()`** function returns the sum of the elements along the main diagonal of an array. The array must have at least two dimensions. If the array used has more than two dimensions then axes need to be specified to determine the 2D sub-array whose traces are returned.
+
+## Syntax
+
+```pseudo
+numpy.trace(a, offset=0, axis1=0, axis2=1, out=None)
```

Comment:
```suggestion
numpy.trace(a, offset=0, axis1=0, axis2=1, out=None)
```
Including a type parameter would make this syntax more accurate though it is not a necessary change.